test_scripts/API/RemoveUrlParameterMaxLength/001_OnSystemRequest.lua

```diff
@@ -0,0 +1,48 @@
+---------------------------------------------------------------------------------------------------
+-- Proposal: https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0200-Removing-URL-Param-Max-Length.md
+--
+-- Description: Check processing of OnSystemRequest notification with different length of url
+--
+-- In case:
+-- 1. HMI sends OnSystemRequest notification with out of lower bound value in url parameter
```

Comment:
`out of lower bound value in url parameter` -> length of url is less than min value